diagnostic.md

```diff
@@ -6,36 +6,46 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    - The Ember Router tells the parses the requests from the client and calls
+    functions that will display react and render things according to the
```

Comment:
I'm not sure exactly what you're trying to say about the router. It does parse the request and calls the Route.

We specify route definitions inside the Application Router.

In a Route, we specify how to load model data using the model hook.